packages/firestore/lib/index.d.ts

```diff
@@ -1969,6 +1969,25 @@ export namespace FirebaseFirestoreTypes {
      * ```
      */
     clearPersistence(): Promise<void>;
+    /**
```

Comment:
This seems like a simply typo in the docs (but you included docs with your PR! :trophy:  :smile: )

```suggestion
```